test/reload_system.bats

```diff
@@ -60,3 +70,33 @@ function expect_log_failure() {
 	# then
 	expect_log_failure
 }
+
+@test "system registries should succeed with new registry without reload" {
+	# given
+	cat << EOF > "$CRIO_CONFIG_DIR/00-automaticReload.conf"
+[crio.image]
+automatically_reload_registries = true
+EOF
+	start_crio
+	replace_config "log_level" "debug"
```

Comment:
Why is this done after, not before starting crio?

Reply:
I added that in a drop in config.